mmf/configs/defaults.yaml

```diff
@@ -4,7 +4,7 @@ config_version: 1.0
 # Configuration for training
 training:
     # Name of the trainer class used to define the training/evalution loop
-    trainer: base_trainer
+    trainer: mmf_trainer
```

Comment:
Can it be just called `mmf` instead of `mmf` trainer? Since, we are here, we should separately discuss how to provide options for trainers. Something like trainer attributes.